app/controllers/api/v1/concerns/ingest_aware.rb

```diff
@@ -26,6 +26,15 @@ module IngestAware
         def validate_scp_user_agent!
           scp_package_headers = extract_scp_user_agent_headers(request)
           if scp_package_headers.any?
+            # log usage to Mixpanel
+            mixpanel_log_props = {
+                user_agent: request.headers['User-Agent'],
+                appFullPath: request.fullpath
+            }
+            if api_user_signed_in?
+              MetricsService.merge_identities_in_mixpanel(current_api_user) if !@user.registered_for_firecloud
```

Comment:
Let's move the `if !registered` check to inside the merge_identities method, so you don't have to duplicate it below.  I think it also makes sense there because it's because of the external metrics service dependencies that we need it, not something related to ingest/omniauth

Reply:
After talking with Eric, we realized that the logic is backwards - we should only ever call `/identify` in Bard if the user _is_ registered for Terra, not the inverse.

To the point, however, I can move it in there at the end and skip the call the Bard if the user isn't registered.

Reply:
9615ad1